gato/gatos/GuinaifenGato.py

```diff
@@ -51,37 +48,14 @@ def deploy(self, team: list["ABaseGato"]):
         # Otherwise, apply buff
         buff_amount = 20 if self.eidolon < 6 else 25
         for g in team:
-            # incrase max energy and max mood as needed
-            max_energy_buff = max(g.energy + buff_amount - g.max_energy, 0)
-            max_mood_buff = max(g.mood + buff_amount - g.max_mood, 0)
-            g.max_energy += max_energy_buff
-            g.max_mood += max_mood_buff
-
-            # record max stats increases so that we can reset them at claim time
-            self.max_stats_buffs.append((g, max_energy_buff, max_mood_buff))
-
             # increase energy and mood
-            g.add_energy(buff_amount)
-            g.add_mood(buff_amount)
+            g.add_energy(buff_amount, allow_overflow=True)
+            g.add_mood(buff_amount, allow_overflow=True)
+        self.recent_stats_buff_ts.append(now)
```

Comment:
Also found a bug from before 😢 
Never added the timestamp in for 2 times a day cap